core/Context.cc

```diff
@@ -77,7 +77,11 @@ GlobalSubstitution::GlobalSubstitution(const GlobalState &from, GlobalState &to,
     : toGlobalStateId(to.globalStateId) {
     Timer timeit(to.tracer(), "GlobalSubstitution.new", from.creation);
     ENFORCE(toGlobalStateId != 0, "toGlobalStateId is only used for sanity checks, but should always be set.");
-    ENFORCE(from.symbols.size() == to.symbols.size(), "Can't substitute symbols yet");
+    ENFORCE(from.classAndModules.size() == to.classAndModules.size(), "Can't substitute symbols yet");
```

Comment:
Hmm, did you merge in changes from master? This is a diff from a change I recently merged to master.

Reply:
this looks like a bad rebase. I will fix!